src/System.Net.Http/src/Resources/Strings.resx

```diff
@@ -198,8 +198,8 @@
   <data name="net_http_client_send_error" xml:space="preserve">
     <value>An error occurred while sending {0}. {1}</value>
   </data>
-  <data name="net_http_client_send_canceled" xml:space="preserve">
-    <value>Request for {0} was canceled.</value>
+  <data name="net_http_client_send_cancelled" xml:space="preserve">
+    <value>Request for {0} was cancelled.</value>
```

Comment:
Yes.  We've tried to standardized on one 'l' (except in "cancellation", which has two 'l's), e.g. "OperationCanceledException".  There was some usage in the framework that predated that, though.